client/src/App.vue

```diff
@@ -209,7 +209,7 @@ export default class App extends Mixins(SessionTimerMixin) {
             const { nrId } = newRequestModule
             // Cancel the NR using the rollback endpoint if we were processing a NEW NR
             await newRequestModule.rollbackNameRequest({ nrId, action: RollbackActions.CANCEL })
-            paymentModule.togglePaymentModal(false)
+            await paymentModule.togglePaymentModal(false)
             // Direct the user back to the start
             await this.resetAppState()
```

Comment:
I think this missing "await" (also below) may be causing some app state to be reset before other things are complete.